apps/web-giddh/src/app/reports/components/reverse-charge-report-component/reverse-charge-report.component.html

```diff
@@ -11,125 +11,291 @@
+                        <div class="total-due-sale total-details-box due-box border-none">
+                            <h5>
+                                {{ localeData?.total_tax_amount }}
+                                <div>
+                                    <span class="d-inline-flex number-total">
+                                        <amount-field
+                                            [amount]="reverseChargeReportResults?.totalTaxAmount"
+                                            [currencySymbol]="activeCompany?.baseCurrencySymbol"
+                                            [currencyCode]="activeCompany?.baseCurrency"
+                                        >
+                                        </amount-field>
+                                    </span>
+                                </div>
+                            </h5>
+                        </div>
+                    </div>
                 </div>
             </div>
-            <div class="top-content" [hidden]="!reverseChargeReportResults?.items?.length">
-                <div class="d-flex w-100 justify-content-between overflow-x-auto">
-                    <div class="total-due-sale total-details-box opening-box border-none mr-r05">
-                        <h5>
-                            {{ localeData?.total_taxable_value }}
-                            <div>
-                                <span class="d-inline-flex number-total">
-                                    <amount-field
-                                        [amount]="reverseChargeReportResults?.totalTaxableValue"
-                                        [currencySymbol]="activeCompany?.baseCurrencySymbol"
-                                        [currencyCode]="activeCompany?.baseCurrency"
-                                    >
-                                    </amount-field>
-                                </span>
+
+            <div class="invoice-preview table-responsive table-header-sticky overflow-visible">
+                <table mat-table matSort [dataSource]="reverseChargeReportResults?.items">
+                    <!-- Index Column -->
+                    <ng-container matColumnDef="index">
+                        <th mat-header-cell *matHeaderCellDef>#</th>
+                        <td mat-cell *matCellDef="let element; let i = index" class="text-center">
+                            {{ (reverseChargeReportGetRequest.page - 1) * paginationLimit + (i + 1) }}
+                        </td>
+                    </ng-container>
+
+                    <!-- Entry Date Column -->
+                    <ng-container matColumnDef="entryDate">
+                        <th mat-header-cell *matHeaderCellDef mat-sort-header (click)="sortReverseChargeList('entryDate')">
+                            {{ localeData?.entry_date }}
+                        </th>
+                        <td mat-cell *matCellDef="let element">{{ element.entryDate }}</td>
```

Comment:
```suggestion
                        <td mat-cell *matCellDef="let element">{{ element?.entryDate }}</td>
```
Correct in over all file

Reply:
Done